src/components/Entry/Card/index.tsx

```diff
@@ -98,6 +73,7 @@ const EntryCard = ({
           <HighlightedText text={name || ''} textToHighlight={search} />
         </Link>
       }
+      subHeader={<>{subtitleFullName}</>}
       footer={
         <>
           {entryDB.toLowerCase() === 'interpro' ? (
```

Comment:
I don't think we need to display the entry type for InterPro entries, since the icon in the header represents the type already.

Instead the accession can be displayed on the left (for both InterPro entries and member database signatures). And the integration status can be displayed on the right (member database signatures only).